kimchi/src/linearization.rs

```diff
@@ -310,19 +325,14 @@ pub fn linearization_columns<F: FftField + SquareRootField>(
 
```

Comment:
Don't you have to insert the Index for any possible new gate added?

Reply:
AFAIK this is something that is new that has happened since I branched from master.  It will be implemented once updated against master.